### PR TITLE
ISPN-5809 - View of cluster events

### PR DESCRIPTION
**Pull request by pzapataf**

### CODE REVIEW

File: src/main/webapp/clusters-view-physical/clusters-view.controller.js [hunk 1]
```diff
@@ -5,9 +5,32 @@ angular.module('managementConsole')
...
+                       $scope.gridEvents.push(event);
+                       });
+                  }
+               );
+           }
+
+          $scope.refreshGridEvents = function() {
+                $scope.gridEvents = [];
+                angular.forEach($scope.clusters, function(cluster){
+                  $scope.refreshClusterEvents(cluster, 10);
```
Comment (vblagoje) [inline src/main/webapp/clusters-view-physical/clusters-view.controller.js]: @pzapataf make 10 a variable somewhere so we can extract/configure it elsewhere if needed

File: src/main/webapp/clusters-view-physical/clusters-view.html [hunk 2]
```diff
@@ -28,7 +28,16 @@
       <div class="sidebar-header sidebar-header-bleed-left sidebar-header-bleed-right">
         <h2 class="h5">Latest status Events:</h2>
       </div>
-      <ul class="list-group">
+      <ul class="list-group"  ng-if="gridEvents.length > 0" >
+        <li class="list-group-item" ng-repeat="event in gridEvents | orderBy:'when'">
+          <h3 class="list-group-item-heading">
+            <span ng-if="event.level=='WARN'"  class="icon-class pficon-warning-triangle-o"></span>
```
Comment (vblagoje) [inline src/main/webapp/clusters-view-physical/clusters-view.html]: we can later dress up these events a bit and use a function here i.e isWarn(), isInfo() etc

File: src/main/webapp/clusters-view-physical/clusters-view.html [hunk 3]
```diff
@@ -28,7 +28,16 @@
...
+            <span ng-if="event.level=='INFO'"  class="icon-class pficon-info"></span>
+            <span ng-if="event.level=='ERROR'" class="pficon-error-circle-o"></span>
+            {{event.message}}.</h3>
+          <small>{{event.when | date:'yyyy-MM-dd HH:mm:ss Z'}}</small><br>
```
Comment (vblagoje) [inline src/main/webapp/clusters-view-physical/clusters-view.html]: nice, you already know how to use filters  :-)

File: src/main/webapp/clusters-view/clusters-view.controller.js [hunk 4]
```diff
@@ -12,8 +12,77 @@ angular.module('managementConsole')
...
+
+            $scope.offlineSites = {};
+            $scope.onlineSites  = {};
+            $scope.mixedSites   = {};
+
+            //
+            // Updates the map of remote site status
+            //
+            $scope.refreshRemoteSitesStatus = function(cluster) {
+              var resourcePathCacheContainer = cluster.domain.getFirstServer().getResourcePath()
```
Comment (vblagoje) [inline src/main/webapp/clusters-view/clusters-view.controller.js]: @tristantarrant told me we'll probably have to replace this call with domain.getCoordinator() eventually

**Comment by vblagoje** [comment]
Great stuff @pzapataf If you want to experiement just a bit more make a events-service.js that will simplify your controller. Now we have a lot of logic in controllers that we have to slowly move into services. You can do it now, if you have time and desire. If now, I'll do it soon enough across the entire codebse.

**Comment by tristantarrant** [comment]
This has already been merged, closing

**Pull request closed without merging**